lib/datasource/index.spec.ts

```diff
@@ -91,4 +91,17 @@ describe('datasource/index', () => {
     });
     expect(res.sourceUrl).toEqual('https://github.com/Jasig/cas');
   });
+  it('applies replacements', async () => {
+    npmDatasource.getPkgReleases.mockResolvedValue({
+      releases: [{ version: '1.0.0' }],
+    });
+    const res = await datasource.getPkgReleases({
+      datasource: datasourceNpm.id,
+      depName: 'abc',
+      replacementName: 'def',
+      replacementVersion: '2.0.0',
+    });
```

Comment:
@viceice any ideas about this? I can't get the mock to return correctly.

Reply:
🤔 I think it needs debugging.  Maybe i have some time tomorrow.

Reply:
found it

Reply:
`global` strikes again!